src/initialize-web.ts

```diff
@@ -0,0 +1,34 @@
+/**
+ * Copyright 2020 Google Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Puppeteer } from './common/Puppeteer.js';
+
+export const initializePuppeteerWeb = (packageName: string): Puppeteer => {
+  const isPuppeteerCore = packageName === 'puppeteer-core';
+
+  // puppeteer-core ignores environment variables
+  return new Puppeteer(
+    // Product root directory is undefined as we're not concerned about
+    // downloading and installing browsers in the web environment
+    undefined,
+    // Preferred revision is undefined as we will use the browser we are running in.
```

Comment:
Ah, this gives me another opportunity to share my favorite tech writing tip ever: 

> Avoid “will”.

http://blog.stoyanstefanov.com/technical-writing-checklist/#:~:text=%22will%22

It’s so simple, but it really works — it's almost always better to remove it. Just wanted to share in case you like it too :)